README.md

```diff
@@ -11,65 +11,85 @@
 1. [License](#license)
 
 ## Background
-This repository is a collection of transformers to be used along with VDB as a plugin to fetch, transform and persist log events and storage slots of
-specified MCD contracts.
+This repository is a collection of transformers to be used along with VDB as a plugin to fetch, transform and persist log events and storage slots of specified MCD contracts.
 
 ## Install
-### Dependencies
+
+### Docker Setup
+
+This repository provides a docker-compose setup to manage the multiple container-based deployment for you, if at all possible you should use that setup.
+
+#### Dependencies
+ - Docker (the rest is handled by docker-compose)
+
+#### Database Initialization
+
+The database needs to be initialized once before bringing up the system. That can be by running, from the root directory, 
+
+```bash
+docker-compose -f dockerfiles/docker-compose.yml run execute ./run_migrations.sh
```

Comment:
Just to double check - have you run this locally and verified the migrations run before the containers run their own migrations? Asking because I've seen the migrations take a long time (~3 minutes) before

Reply:
Just double-checked and when that command is run locally it only starts the postgres image, and the execute image. The others aren't run, and the execute image actually shuts down when run_migration is done, because it overrides the startup command.